$OpenBSD: patch-include_data_h,v 1.5 2012/11/02 13:59:21 dcoppa Exp $

From e7a4580c5f5f895df2990dae0f26a8662d58d80f Mon Sep 17 00:00:00 2001
From: Michael Stapelberg <michael@stapelberg.de>
Date: Mon, 29 Oct 2012 15:41:16 +0000
Subject: Bugfix: force rendering when the parent's orientation changed

Otherwise, the split indicator might not be refreshed even though
it should be.

--- include/data.h.orig	Wed Sep 19 18:08:09 2012
+++ include/data.h	Fri Nov  2 14:32:07 2012
@@ -135,6 +135,7 @@ struct deco_render_params {
     Rect con_deco_rect;
     uint32_t background;
     bool con_is_leaf;
+    orientation_t parent_orientation;
 };
 
 /**
